source/Core/Controller/BaseController.php

```diff
@@ -867,6 +875,8 @@ public function getSidForWidget()
         if (!$oSession->isActualSidInCookie()) {
             return $oSession->getId();
         }
+
+        return null;
```

Comment:
in the line above: you added "return null;" which I would not recommend for php developers to do explicitly as this is the default and adds no value. For perl or other languages it's another story.

Reply:
solved in https://github.com/OXID-eSales/oxideshop_ce/pull/676/commits/dc7fde207e69ebd01aff5227f7482aa48771b68e and https://github.com/OXID-eSales/oxideshop_ce/pull/676/commits/4e2501d1fa6d46697c12df0114a4047cead6787e